package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "floeproject.org",
+  "version": "1.0.0",
+  "description": "FLOE Project Website",
+  "main": "index.js",
+  "scripts": {
+    "build": "eleventy",
+    "eleventy": "eleventy --serve",
+    "eleventyport": "eleventy --serve --port=9778"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/fluid-project/floeproject.org.git"
+  },
+  "keywords": [
+    "Floe",
+    "Project",
+    "Flexible",
+    "Learning",
+    "for",
+    "Open",
+    "Education"
+  ],
+  "author": "OCAD University <idrc@ocadu.ca>",
+  "contributors": [
+    "Sachin Chopra <schopra@mt.iitr.ac.in> (www.github.com/sachin10101998)"
+  ],
```

Comment:
You should remove the contributors section. Mostly it's just hard to maintain. We've started using a centralized AUTHORS.md file and copyright. See an example from Infusion:

- https://github.com/fluid-project/infusion/blob/master/AUTHORS.md
- https://github.com/fluid-project/infusion/blob/master/Gruntfile.js

Reply:
Fixed